src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Value/AttributeMatch.php

```diff
@@ -31,33 +31,29 @@ class AttributeMatch implements JsonSerializable
      */
     private $remarks = '';
     /**
-     * @var string
+     * @var Attribute
      */
-    private $name = '';
+    private $local;
     /**
-     * @var string
+     * @var Attribute
      */
-    private $value = '';
+    private $remote;
 
     /**
-     * @param string $name
-     * @param string[] $value
+     * @param Attribute $local
+     * @param Attribute $remote
      * @param bool $valid
      * @param string $remarks
-     * @throws \Assert\AssertionFailedException
      */
-    public function __construct($name, $value, $valid, $remarks)
+    public function __construct(Attribute $local, Attribute $remote, $valid, $remarks)
     {
-        Assert::string($name, 'name should be string');
-        Assert::isArray($value, 'value should be an array');
-        Assert::allString($value, 'values should be string');
         Assert::boolean($valid, 'valid should be boolean');
         Assert::string($remarks, 'remarks should be string');
 
-        $this->name = $name;
+        $this->local = $local;
+        $this->remote = $remote;
```

Comment:
I've already expressed my dislike of the local and remote terminology. 

But I think you can't simply call these attributes `local` and `remote`, at least add `attribute`. `getLocalAttribute` would be so much more expressive in my opinion.

Reply:
Done!